sys-block/storcli/storcli-7.2405.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit rpm
+
+# Upstream is still using strange version numbers
+MY_PV="00${PV}.0000.0000"
+MY_P="${PN}-${MY_PV}"
+MY_PN="STORCLI_SAS3.5_P25"
+DESCRIPTION="MegaRAID StorCLI (successor of the MegaCLI)"
+HOMEPAGE="https://www.broadcom.com/support/download-search?dk=storcli"
+SRC_URI="https://docs.broadcom.com/docs-and-downloads/host-bus-adapters/host-bus-adapters-common-files/sas_sata_nvme_12g_p25/${MY_PN}.zip -> ${P}.zip"
+
+LICENSE="Avago LSI BSD"
+SLOT="0"
+KEYWORDS="-* amd64"
+BDEPEND="app-arch/unzip"
+
+S="${WORKDIR}"
+
+QA_PREBUILT="/opt/MegaRAID/storcli/storcli64"
+
+src_unpack() {
+	default
+	rpm_unpack ./${MY_PN}/univ_viva_cli_rel/Unified_storcli_all_os/Linux/${MY_P}-1.noarch.rpm
+	unzip ./${MY_PN}/univ_viva_cli_rel/Unified_storcli_all_os/JSON-Schema/JSON_SCHEMA_FILES.zip
+}
+
+src_prepare() {
+	default
+
+	sed -i -e 's|/home/|/usr/share/storcli/|g' JSON-Schema/*.json
```

Comment:
find JSON-Schema/ -name *.json -exec sed -i -e 's|/home/|/usr/share/storcli/|g' {} \;

I see a more complex syntax

Reply:
Probably due to bad personal experiences in the past, I trust `find -type f -name *.json` more than globbing like that :I 

Anyway if you want to go into semantics, `find ... {} +` should be cheaper, but then again it depends how many files it will have to modify.

EITHER way you'll have to append the command with `|| die", 
https://devmanual.gentoo.org/ebuild-writing/error-handling/index.html#the-die-function